libcryptsetup-rs-sys/header.h

```diff
@@ -1 +1,56 @@
-#include <libcryptsetup.h>
+#include "libcryptsetup.h"
+
+const unsigned int crypt_activate_readonly = CRYPT_ACTIVATE_READONLY;
```

Comment:
Just a minor suggestion. Those flags are strictly 32 bit, so I'd use:

```
const uint32_t crypt_activate_readonly = CRYPT_ACTIVATE_READONLY;
(...)
```
Or did I miss that's the idea behind the workaround, in fact?

Reply:
I was more leaving it that way because they were `unsigned int`s previously before the addition of the `U32INT_C` macro on the C side. I don't see a compelling reason not to switch them to `uint32_t` if that's what you'd prefer. My understanding is that `c_uint` is the same as `u32` on all of Rust's supported platforms so practically it shouldn't make a difference.

Reply:
Ah, thanks! That makes sense - sorry, I'm not that familiar with libcryptsetup and I hadn't realised that this is how they were previously defined.